### PR TITLE
Increase max allowed package to avoid errors

### PR DESCRIPTION
**Pull request by Fulmenef**

### CODE REVIEW

File: mysql/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM mysql:5.7
+
+# Automatically start the SSH agent when a new session is created
```
Comment (ajardin) [inline mysql/Dockerfile]: Wrong copy/paste I guess... 😉

File: mysql/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+RUN echo 'max_allowed_packet=256M' >> /etc/mysql/conf.d/docker.cnf
```
Comment (ajardin) [inline mysql/Dockerfile]: Could you add a newline at the end of file?

File: mysql/Dockerfile (hunk 1 above)
Comment (Fulmenef) [reply mysql/Dockerfile]: Indeed, it's fixed thanks :)

File: mysql/Dockerfile (hunk 2 above)
Comment (Fulmenef) [reply mysql/Dockerfile]: Done :)

**Pull request merged**